Decred Wallet/Constants/Constants.swift

```diff
@@ -9,21 +9,7 @@
 import Foundation
 import UIKit
 
-struct GlobalConstants {
-    //MARK: - Colors
-    // todo move to UIColor.swift
-    struct Colors {
-        static let orangeColor = UIColor(hex: "fd714a")
-        static let navigationBarColor = UIColor(hex: "689F38")
-        static let lightGrey = UIColor(red: 236.0/255.0, green: 238.0/255.0, blue: 241.0/255.0, alpha: 1.0)
-        static let lightBlue =  UIColor(red: 206.0/255.0, green: 238.0/255.0, blue: 250.0/255.0, alpha: 1.0)
-        static let separaterGrey = UIColor(red: 224/255, green: 224/255, blue: 224/255, alpha: 1.0)
-        static let greenishGrey = UIColor(hex: "F1F8E9")
-        static let black = UIColor(hex: "000000")
-        static let TextAmount = UIColor(hex: "#091440")
-        static let displayAamount = UIColor(hex: "617386")
-    }
-    
+struct GlobalConstants {    
```

Comment:
There seem to be some extra whitespaces added to this line, it should not have been modified at all.

Reply:
resolved in 649c901